SIL.Core.Desktop/SIL.Core.Desktop.csproj

```
@@ -20,7 +20,7 @@
   <ItemGroup>
     <ProjectReference Include="..\SIL.Core\SIL.Core.csproj" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
```

I'd do this instead to ensure it only runs when targeting framework, otherwise if this ever references more recent dotnet the current setup will include this block
```suggestion
  <ItemGroup Condition=" $(DefineConstants.Contains('NETFRAMEWORK')) ">
```

---

Done